azureLogIngestion/adapter/index.ts

```diff
@@ -28,15 +28,11 @@ export default class Adapter {
      * There may be situations where a message corresponds to more than one
      * type of telemetry. In this case, the switch/case may not make sense.
      */
-    processMessages(messages: string): void {
```

Comment:
Why this change? Don't we want the function doing as little as possible?

Reply:
Because the structure of the messages is different than what the code was expecting. It was erroring with undefined until I made these changes.